rfd/0036/queue/README.md

```diff
@@ -53,13 +52,12 @@ The following table describes whether tasks can be executed concurrently, sequen
 
 Tasks reading down the chart are those already in the queue; tasks reading across the chart are those to be added.
 
-|                      | Add `start` | Add `stop` | Add `scale` | Add `reprovision` | Add `freeze` |
-|----------------------|-------------|------------|-------------|-------------------|--------------|
-| `start` active       | Reject      | Terminate  | FIFO        | FIFO              | Terminate    |
-| `stop` active        | Reject      | Reject     | Reject      | Reject            | Terminate    |
-| `scale` active       | FIFO        | Terminate  | Reject      | Concurrent        | Terminate    |
-| `reprovision` active | FIFO        | Terminate  | Concurrent  | Terminate         | Terminate    |
-| `freeze` active      | Terminate   | Terminate  | Terminate   | Terminate         | Reject       |
+|                      | Add `start` | Add `stop` | Add `scale` | Add `reprovision` |
+|----------------------|-------------|------------|-------------|-------------------|
+| `start` active       | Reject      | Terminate  | FIFO        | FIFO              |
+| `stop` active        | Reject      | Reject     | Reject      | Reject            |
+| `scale` active       | FIFO        | Terminate  | Reject      | Concurrent        |
+| `reprovision` active | FIFO        | Terminate  | Concurrent  | Terminate         |
```

Comment:
One edge case I can think of with parallel execution of `scale` and `reprovision` tasks is scaling down during a `reprovision`:

I have 3 copies of a service running. I tell Triton to `reprovision` them all at once. While that's happening, I scale down to two.